tests/cases/compiler/parametersSyntaxErrorNoCrash1.ts

```diff
@@ -0,0 +1,6 @@
+// @strict: true
+// @noEmit: true
+
+function identity<T>(arg: T: T {
+    return arg;
+}
```

Comment:
It's pretty important that we don't have any extra characters here (no newline). For that reason, it might be better to move this test somewhere else where I'd have improved control over source text. I'm not sure where such tests are kept - I could use a hint :)

Reply:
Leaving this is fine IMO; we don't format tests. The only hazard is if someone has the editor flag enabled to add one (or uses one of the many editors that adds it, correctly tbh). Could leave a note, though.

Alternatively, you could write it as a unit test; it wouldn't be a bad idea to make a new file for these kinds of crashes in the unittest folder.